source/arrayable.d.ts

```diff
@@ -23,4 +23,4 @@ bundle('src/index.js', ['dist/index.cjs', 'dist/index.mjs']);
 
 @category Array
 */
-export type Arrayable<T> = T | readonly T[];
+export type Arrayable<T> = T | T[];
```

Comment:
Add a TODO comment that links to the TS issue, so we someday can add back `readonly`.